apps/web/components/shared/project-card-variants.tsx

```
@@ -0,0 +1,384 @@
+            </button>
+
+            <div className="space-y-4">
+                <div className="mb-4">
+                    <div className="flex items-center gap-2 mb-4">
+                        <Gift className="w-5 h-5 text-purple-500" />
+                        <span className="font-semibold text-gray-700">NFT Rewards</span>
+                    </div>
+                </div>
+
+                {nftTiers.map((tier, index) => (
+                    <div key={index} className="p-4 bg-gray-50 rounded-lg">
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)